neoexchange/astrometrics/tests/test_sources_subs.py

```diff
@@ -415,7 +416,36 @@ def test_page_down(self):
         self.assertEqual(expected_targets, targets)
 
 
-class TestFetchGoldstoneTargets(TestCase):
+class TestFetchGoldstoneCSV(SimpleTestCase):
+
+    def setUp(self):
+        self.test_file = os.path.join('astrometrics', 'tests', 'test_goldstone_page.csv')
+
+    def test_basic(self):
+        expected_length = 7
+        expected_columns = ['number', 'name', 'start (UT)', 'end (UT)', 'OCC', 'Updated 2021 Dec 17']
+
+        table = fetch_goldstone_csv(self.test_file)
+
+        self.assertEqual(expected_length, len(table))
+        self.assertEqual(expected_columns, table.colnames)
+
+    def test_missing_file(self):
+        expected_length = None
+
+        table = fetch_goldstone_csv('/foo/bar')
+
+        self.assertEqual(expected_length, table)
+
+    def test_missing_url(self):
+        expected_length = None
+
+        table = fetch_goldstone_csv('https://foo.bar.com/nothere.csv')
```

Comment:
Excited for this domain to be claimed and populated in the future. :P

Reply:
It needed to be a properly formatted domain otherwise it failed differently inside _astropy_s `request`-wrapped internals... Slightly surprised someone hasn't grabbed it already...